### ISSUE TITLE
Web SPA use Blazor

### ISSUE THREAD

**Issue by geffzhang** [dotnet-architecture/eShopOnDapr#18]
Blazor server is more reflected in the combination with Dapr, and blazor wasm is a better alternative to js' dotnet web project ，Do you plan to add blackzor as a web ui?

**Comment by amolenk** [comment]
I'm looking at it, but having good Dapr support in the back-end services is the main priority right now.

**Comment by Shawn-Fan** [comment]
Interesting, Do you have any plan on back-end services ? I mean some works finished in the current project but which service bus, pubsub(based on kafka/rabbitmq ...) will you work in the future?

**Comment by geffzhang** [comment]
Discover that the dotnet team is writing a dapr for dotnet developer e-book https://github.com/dotnet/docs/tree/ebook-dapr-for-net-devs/docs/architecture/dapr-for-net-developers The current progress is Chapter VIII， This e-book code example looks like this repo

**Comment by amolenk** [comment]
Yes, we're currently writing a *Dapr for .NET Developers* ebook which will be released alongside the 1.0 release of Dapr.

The sample code in this repo is a version of eShopOnContainers modified to show the use of the Dapr building blocks, and is used extensively in the ebook. 

Because the code uses Dapr, it's very easy to switch infrastructure by simply configuring different components. For example, you can choose any component from this list for pub/sub: https://docs.dapr.io/operations/components/setup-pubsub/supported-pubsub/

**Comment by Shawn-Fan** [comment]
I try to download the pdf document, but the link is to another one(https://dotnet.microsoft.com/download/e-book/cloud-native-azure/pdf), any idea?

**Comment by robvet** [comment]
Shawn... sounds like you’re trying to download a .PDF, but getting a different one than you’re requesting? Which .PDF are you trying to download and which .PDF are you seeing?

Thanks

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Shawn-Fan** [comment]
@robvet 
I try to download "dapr-for-net-developers" .PDF file from the link mentioned above but I got the "cloud-native-azure" ebook (although "cloud-native-azure" is an excellent one). I want to know the roadmap for the messaging development of the project if any.
Thanks!
